llvm/unittests/tools/llvm-mca/MCATestBase.h

```diff
@@ -73,9 +74,11 @@ class MCATestBase : public ::testing::Test {
   /// Utility function to run MCA with (nearly) the same configuration as the
   /// `llvm-mca` tool to verify result correctness.
   /// This function only displays on SummaryView by default.
-  virtual Error runBaselineMCA(json::Object &Result, ArrayRef<MCInst> Insts,
-                               ArrayRef<mca::View *> Views = {},
-                               const mca::PipelineOptions *PO = nullptr);
+  virtual Error
+  runBaselineMCA(json::Object &Result, ArrayRef<MCInst> Insts,
+                 ArrayRef<mca::View *> Views = {},
+                 const mca::PipelineOptions *PO = nullptr,
+                 SmallVector<std::pair<StringRef, StringRef>> Descs = {});
```

Comment:
please use ArrayRef here: https://llvm.org/docs/ProgrammersManual.html#llvm-adt-smallvector-h (see the "Note")